### ISSUE TITLE
After one day of training, the perplexity and accuracy of validation set remained unchanged

### ISSUE THREAD

**Issue by lijun20** [OpenNMT/OpenNMT-py#965]
Validation perplexity: 56.168
Validation accuracy: 35.9623

train parameter：
encoder_type=transformer  and  decoder_type=transformer
the accuracy and perplexity of validation remains unchanged. How can I solve this problem ?

**Comment by vince62s** [comment]
ask your question on the forum by  providing context and command lines, thanks.
